src/lib/services/graphql/graphql.service.ts

```diff
@@ -419,28 +447,34 @@ export class GraphqlService {
   }
 
   private monitorWithOfflineEvent(): void {
-    window.addEventListener('offline', this.sendPingWithThisBound);
+    if (typeof window !== 'undefined') {
+      window.removeEventListener('offline', this.sendPingWithThisBound);
```

Comment:
Why to remove and add again? To avoid adding listener twice?

Reply:
Currently offline listener is added whenever websocket has opened, and never cleaned up. Would be good to avoid adding it multiple times